packages/ui/src/molecules/Dropdown/DropdownItem.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ButtonHTMLAttributes } from 'react'
+import React, { useImperativeHandle, forwardRef, useRef, useState } from 'react'
+
+import { useDropdown } from './hooks/useDropdown'
+
+export interface DropdownItemProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const DropdownItem = forwardRef<HTMLButtonElement, DropdownItemProps>(
+  function Button(
+    { children, onClick, testId = 'store-dropdown-item', ...otherProps },
+    ref
+  ) {
+    const {
+      dropdownItemsRef,
+      selectedDropdownItemIndexRef,
+      close,
+    } = useDropdown()
+
+    const [dropdownItemIndex, setDropdownItemIndex] = useState(0)
+    const dropdownItemRef = useRef<HTMLButtonElement>()
+
+    const addToRefs = (el: HTMLButtonElement) => {
+      if (el && !dropdownItemsRef?.current.includes(el)) {
```

Comment:
Is this `includes` working as expected? 
`Includes` is better for primitives, but with objects, I think the `.some()` function is the best option.